rudder-core/src/main/resources/ldap/bootstrap.ldif

```diff
@@ -163,7 +163,7 @@ isSystem: TRUE
 dn: techniqueCategoryId=Active Techniques,ou=Rudder,cn=rudder-configuration
 objectClass: techniqueCategory
 objectClass: top
-cn: Root of active techniques's library
+cn: Root of active techniques library
```

Comment:
Simpler, shorter, go let's go (only concern would be about the T of techniqes to be capitalized or not) I guess it should always be capitalized